sdk/core/azure-core/vcpkg/vcpkg.json

```diff
@@ -75,7 +75,8 @@
         {
           "name": "azure-core-cpp",
           "default-features": false
-        }
+        },
+        "wil"
```

Comment:
To confirm, @LarryOsterman, this `wil` dependency is **only** needed for the winhttp feature?

Reply:
That is correct - "wil" stands for "windows implementation library" and is windows-only. We only use it for winhttp to deal with winhttp's abuse of the `void *` type.